test/uk/gov/hmrc/cgtpropertydisposals/connectors/SubscriptionConnectorImplSpec.scala

```diff
@@ -57,11 +58,85 @@ class SubscriptionConnectorImplSpec extends WordSpec with Matchers with MockFact
 
   "SubscriptionConnectorImpl" when {
 
-    "handling request to subscribe" must {
+    implicit val hc: HeaderCarrier = HeaderCarrier()
+    val expectedHeaders            = Map("Authorization" -> s"Bearer $desBearerToken", "Environment" -> desEnvironment)
+
+
+    "handling request to subscribe for individuals" must {
+
+      val subscriptionDetails = SubscriptionDetails(
+        Right(Name("name", "surname")),
+        "email",
+        UkAddress("line1", None, None, None, "postcode"),
+        "sap"
+      )
+
+      val expectedRequest = Json.parse(
+        """
+          |{
+          |  "typeOfPerson" : "1",
+          |  "contactName"  : "name surname",
```

Comment:
sigh - DES data modelling at it's finest.

Reply:
this is subject to change - perhaps we can get a better model when we negotiate the specs with them if we ask nicely enough